packages/DApp/src/config.ts

```diff
@@ -0,0 +1,75 @@
+import { v4 as uuidv4 } from 'uuid'
+import { ChainId, OptimismGoerli, Optimism, Config as DAppConfig, Localhost, Hardhat } from '@usedapp/core'
+
+const version = '0.0.5'
+
+export interface Config {
+  wakuConfig: {
+    environment: 'test' | 'production'
+    wakuTopic: string
+    wakuFeatureTopic: string
+  }
+  daapConfig: DAppConfig
+  statusWalletRequired: boolean
+}
+
+const configs: Record<typeof process.env.ENV, Config> = {
+  localhost: {
+    statusWalletRequired: false,
+    wakuConfig: {
+      environment: 'production',
```

Comment:
non-blocking: I remember saying otherwise, but in retrospect I'd point local to test in order not to accidentally hammer prod peers. Even though if it means switching fleet in app's settings.

Reply:
Same for dev, I think.

Reply:
I guess it also depends on wether we'll deploy new nodes for this or not, ref https://github.com/status-im/community-dapp/issues/23.

Could be decided as follow-up though.